site/source/tutorials/1040_dojo1_dijits/index.md

```diff
@@ -34,7 +34,7 @@ The first step will be to install our dependencies using npm:
 $ npm install
 ```
 
-{% instruction 'Install additional development dependency:' %}
+{% instruction 'Install additional development dependencies:' %}
```

Comment:
It is only one dependency... are you sure you want this to be plural?

Reply:
Fixed!